mula/scheduler/connectors/connector.py

```diff
@@ -40,10 +40,10 @@ def is_host_healthy(self, host: str, health_endpoint: str) -> bool:
         """
         try:
             url = f"{host}/{health_endpoint}"
-            response = requests.get(url, timeout=5)
+            response = httpx.get(url, timeout=5)
             healthy = response.json().get("healthy")
             return healthy
-        except requests.exceptions.RequestException as exc:
+        except httpx.RequestError as exc:
```

Comment:
The httpx equivalent of RequestException is HTTPError. This isn't changes correctly in a lot of places, I haven't added comments for all the others.